src/content/vintage/getting-started/observability/visualization/log-exploration/index.md

```diff
@@ -0,0 +1,149 @@
+---
+linkTitle: How to explore logs with LogQL
+title: Exploring logs with LogQL
+description: Guide explaining how to get explore logs of your management and workload clusterrs stored in Giant Swarm managed Loki.
+menu:
+  main:
+    identifier: getting-started-observability-visualization-exploring-logs
+    parent: getting-started-observability-visualization
+weight: 40
+aliases:
+  - /getting-started/observability/visualization/exploring-logs
+last_review_date: 2024-02-26
+user_questions:
+  - How can I access logs from my installation?
+  - Where can I find workload cluster logs?
+  - Which logs are gathered in my installation?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+---
+
+Logs, which are records of events and activities generated by systems and applications, serve as a rich source of information for administrators, developers, and security professionals. The ability to search through logs is essential to help with troubleshooting and debugging by looking through a detailed history of events, errors and warnings. It is also a valuable source of information in context of performance monitoring, security and compliance.
+
+This guide provides you with the basics to get started with exploring Logs using the powerful capabilities brought by Grafana and Loki.
+
+## How do I access the logs ?
+
+Once you have [access to your management cluster's grafana]({{< relref "/vintage/getting-started/observability/visualization/access" >}}), you should:
+
+1. Go to `Explore` item in the `Home` menu
+![Grafana Explore](loki-explore.png)
+
+2. Select `Loki` datasource on the top left corner
+![Loki datasource](loki-datasource-query.png)
+
+3. Choose how you prefer to build your queries:
+   * `builder` and play with the dropdowns to build your query
+   * `code` to write your query using [LogQL](https://grafana.com/docs/loki/latest/logql/)
+
+__Note__: The live mode feature of Grafana Loki is not available at the moment because we use multi-tenancy (c.f. https://github.com/grafana/loki/issues/9493) 😢
+
+## LogQL basics
+
+Here are a few LogQL basics to help you get started.
+
+### Query anatomy
+
+![Loki Query Anatomy](loki-query-anatomy.png)
+
+### Log stream selectors
+
+Used to filter the log stream by labels
+
+```nohighlight
+=  : equals
+!= : not equals
+=~ : regex matches
+!~ : regex does not match
+```
+
+### Filter operators
+
+Used to filter text within the log stream
+
+```nohighlight
+|= : contains
+!= : does not contain
+|~ : regex matches
+!~ : regex does not match
```

Comment:
```suggestion
Filter operators are used to refine the selection of text within the log stream. They help you narrow down the results by matching specific patterns or excluding certain data.

```nohighlight
|= : This operator is used when the log stream contains a specific string. It returns all log lines that contain the specified string.
!= : This operator is used when the log stream does not contain a specific string. It filters out and excludes all log lines that contain the specified string.
|~ : This operator is used for regex (regular expression) matching. It returns all log lines where the specified regular expression pattern matches.
!~ : This operator is used when the regular expression does not match the log stream. It filters out and excludes all log lines where the specified regular expression pattern matches.
```